extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -46,6 +52,12 @@ class AmpYoutube extends AMP.BaseElement {
     /** @private @const {number} */
     this.height_ = getLengthNumeral(height);
 
+    /** @private @const {!string} */
+    this.frameId_ = 'amp-youtube-' + count++;
```

Comment:
Use `i-amp-youtube-` instead. We don't allow "i-amp" in the user code. Otherwise, this can overlap.